--- beagle-2.1/libhmsbeagle/GPU/kernels/Makefile.am	2014-02-17 12:59:01.000000000 -0800
+++ patch-files/Makefile.am	2015-06-17 10:16:09.568125210 -0700
@@ -39,7 +39,7 @@
 # Compile double-precision kernels
 #
 # 	Compile 4-state model
-	$(NVCC) -o BeagleCUDA_kernels.ptx -ptx -arch compute_13 -DCUDA -DSTATE_COUNT=4 -DDOUBLE_PRECISION \
+	$(NVCC) -o BeagleCUDA_kernels.ptx -ptx -arch compute_20 -DCUDA -DSTATE_COUNT=4 -DDOUBLE_PRECISION \
 		$(srcdir)/kernels4.cu $(NVCCFLAGS) -DHAVE_CONFIG_H $(INCLUDE_DIRS)
 	echo "#define KERNELS_STRING_DP_4 \"" | sed 's/$$/\\n\\/' >> BeagleCUDA_kernels.h
 	cat BeagleCUDA_kernels.ptx | sed 's/\"/\\"/g' | sed 's/$$/\\n\\/' >> BeagleCUDA_kernels.h
@@ -49,7 +49,7 @@
 #
 	for s in $(STATE_COUNT_LIST); do \
 		echo "Making state count = $$s" ; \
-		($(NVCC) -o BeagleCUDA_kernels.ptx -ptx -arch compute_13 -DCUDA -DSTATE_COUNT=$$s -DDOUBLE_PRECISION \
+		($(NVCC) -o BeagleCUDA_kernels.ptx -ptx -arch compute_20 -DCUDA -DSTATE_COUNT=$$s -DDOUBLE_PRECISION \
 			$(srcdir)/kernelsX.cu $(NVCCFLAGS) -DHAVE_CONFIG_H $(INCLUDE_DIRS)) || exit; \
 		echo "#define KERNELS_STRING_DP_$$s \"" | sed 's/$$/\\n\\/' >> BeagleCUDA_kernels.h; \
 		cat BeagleCUDA_kernels.ptx | sed 's/\"/\\"/g' | sed 's/$$/\\n\\/' >> BeagleCUDA_kernels.h; \
